chapter_4/ex03_file/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+# CMakeLists file for the Chapter 4 Example nr. 03: 
+# Installable files/programs
+#
+# SPDX-License-Identifier: MIT
+
+project(
+  ch4_ex03_file
+  VERSION 1.0
+  DESCRIPTION "Chapter 4 Example 03, install file/program"
+  LANGUAGES CXX)
+
+install(FILES ${CMAKE_CURRENT_LIST_DIR}/chapter4_greeter_content TYPE BIN)
+install(PROGRAMS ${CMAKE_CURRENT_LIST_DIR}/chapter4_greeter.py TYPE BIN RENAME chapter4_greeter)
```

Comment:
These were added while I was reviewing, but same applies as in `chapter_4/ex04_directory/CMakeLists.txt`.